install/kubernetes/helm/istio/charts/galley/templates/service.yaml

```diff
@@ -13,7 +13,7 @@ spec:
   ports:
   - port: 443
     targetPort: 9443
-    name: https-validation
+    name: https-valid
```

Comment:
This is not needed anymore, but will keep it as long as this is a WIP for testing purposes, as it was fixed ~1 week ago and not in any k8s release. See https://github.com/kubernetes/kubernetes/issues/84388